lms/djangoapps/teams/static/teams/js/views/edit_team.js

```diff
@@ -6,15 +6,16 @@
             'gettext',
             'js/views/fields',
             'teams/js/models/team',
+            'common/js/components/utils/view_utils',
             'text!teams/templates/edit-team.underscore'],
-        function (Backbone, _, gettext, FieldViews, TeamModel, editTeamTemplate) {
+        function (Backbone, _, gettext, FieldViews, TeamModel, ViewUtils, editTeamTemplate) {
             return Backbone.View.extend({
 
                 maxTeamNameLength: 255,
                 maxTeamDescriptionLength: 300,
 
                 events: {
-                    'click .action-primary': 'createOrUpdateTeam',
+                    'click .action-primary': function(event) {this.disableRun(event, this.createOrUpdateTeam);},
```

Comment:
![screen shot 2015-09-18 at 3 19 25 pm](https://cloud.githubusercontent.com/assets/7373924/9968813/b604b28a-5e18-11e5-8e75-225e33c6f651.png)

This is where I'm running into issues. It is completing successfully though, since in the non-failure case we're redirecting anyways.